README.md

```diff
@@ -1,4 +1,4 @@
-# DremioModule
+# Dremio Module
 
```

Comment:
maybe make sense to add a sentence or two about what is the DremioModules, why do we need it.

Reply:
Added a sentence or two.

Reply:
See current brief explanation